testcases/parser/visitor.go

```diff
@@ -159,7 +159,7 @@ func (v *TestCaseVisitor) VisitMultiArgAggregateFuncCall(ctx *baseparser.MultiAr
 	if ctx.QualifiedAggregateFuncArgs() != nil {
 		args = v.Visit(ctx.QualifiedAggregateFuncArgs()).([]*AggregateArgument)
 	}
-	testcase.FuncName = ctx.Identifier().GetText()
+	testcase.FuncName = strings.ToLower(ctx.Identifier().GetText())
```

Comment:
Are function names in Substrait case sensitive because they're effectively part of a URI?  If so we may want to keep the provided case.  If we think it is wrong to have names that aren't in lowercase we could raise an error.

Reply:
Reverted it.